Flask_app/views/auth.py

```diff
@@ -43,15 +43,15 @@ def post(self):
                 )
                 return jsonify(message="User register successfully")
             except IntegrityError:
-                return jsonify(message="User already exists")
+                return jsonify(message="User already exists"),400
 
         elif request.path == "/users/login":
             email = request.json.get("email" , None)
             password = request.json.get("password" , None)
             user = get_user_by_email(email)
             if user and user.verify_password(password):
                 access_token = create_access_token(identity=user.id,expires_delta=timedelta(minutes=30.0))
-                return jsonify({ "token": access_token, "user_id": user.id})
+                return jsonify({ "token": access_token})
             else:   
```

Comment:
Can remove else block, Just return it outside if block
return jsonify(message="Email or password doesn't match")